CHANGES.rst

```diff
@@ -6,6 +6,8 @@ Changelog
 
 Minor changes:
 
+- Converted all ``format`` strings to ``f-strings`` in cal.py. Ref: #481 [pronoym99]
+
```

Comment:
There would be no new line in between.

Reply:
Oh sorry...I resolved the merge conflict at just the time you edited your comment.